### PR TITLE
fix(core): Fix url in error handelling for the error Trigger

### PR DESCRIPTION
**Pull request by agobrech**
Github issue / Community forum post (link here to close automatically):
https://github.com/n8n-io/n8n/issues/5197

### CODE REVIEW

File: packages/nodes-base/nodes/ErrorTrigger/ErrorTrigger.node.ts [hunk 1]
```diff
@@ -54,7 +54,7 @@ export class ErrorTrigger implements INodeType {
 			items[0].json = {
 				execution: {
 					id,
-					url: `${urlParts.join('/')}/${id}`,
+					url: `${urlParts.join('/')}/workflow/1/${id}`,
```
Comment (csuermann) [inline packages/nodes-base/nodes/ErrorTrigger/ErrorTrigger.node.ts:57]: the hardcoded '1' has a smell... 🤔

Comment (agobrech) [reply packages/nodes-base/nodes/ErrorTrigger/ErrorTrigger.node.ts:57]: This is for the manual execution for testing out the node output. It's not the actual behaviour of the node.

Comment (csuermann) [reply packages/nodes-base/nodes/ErrorTrigger/ErrorTrigger.node.ts:57]: ah, ok. Thanks!

**Review by Joffcom — APPROVED** [review]
Tested and working as expected 👍🏻

**Comment by janober** [comment]
Got released with `n8n@0.212.1`

**Pull request merged**